src/write_http.c

```diff
@@ -489,6 +490,376 @@ static int wh_write (const data_set_t *ds, const value_list_t *vl, /* {{{ */
         return (status);
 } /* }}} int wh_write */
 
+static int notification_meta_data_to_json(char *buffer, size_t buffer_size, /* {{{ */
```

Comment:
Can you please move this code to `src/utils_format_json.c` so other plugins can benefit from it?